<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>931</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculateS2P</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="Calculate_Tab">
      <property name="font">
       <font>
        <family>等线</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="Cal_tab">
       <attribute name="title">
        <string>Calculate</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,3">
            <item>
             <widget class="QGroupBox" name="RL_Label">
              <property name="font">
               <font>
                <family>等线</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>回波损耗</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <widget class="QTableWidget" name="RL_Table">
                 <property name="font">
                  <font>
                   <family>等线</family>
                  </font>
                 </property>
                 <attribute name="horizontalHeaderMinimumSectionSize">
                  <number>20</number>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>94</number>
                 </attribute>
                 <attribute name="verticalHeaderMinimumSectionSize">
                  <number>25</number>
                 </attribute>
                 <attribute name="verticalHeaderDefaultSectionSize">
                  <number>25</number>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="MaxIL_Label">
              <property name="font">
               <font>
                <family>等线</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>边频损耗</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QTableWidget" name="MaxIL_Table">
                 <property name="font">
                  <font>
                   <family>等线</family>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="autoScrollMargin">
                  <number>16</number>
                 </property>
                 <property name="rowCount">
                  <number>0</number>
                 </property>
                 <property name="columnCount">
                  <number>0</number>
                 </property>
                 <attribute name="horizontalHeaderCascadingSectionResizes">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderMinimumSectionSize">
                  <number>20</number>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>94</number>
                 </attribute>
                 <attribute name="verticalHeaderMinimumSectionSize">
                  <number>25</number>
                 </attribute>
                 <attribute name="verticalHeaderDefaultSectionSize">
                  <number>25</number>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="Ripple_Label">
              <property name="font">
               <font>
                <family>等线</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>通带纹波</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <widget class="QTableWidget" name="Ripple_Table">
                 <attribute name="horizontalHeaderMinimumSectionSize">
                  <number>20</number>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>94</number>
                 </attribute>
                 <attribute name="verticalHeaderDefaultSectionSize">
                  <number>25</number>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="AvgIL_Label">
              <property name="font">
               <font>
                <family>等线</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>平均损耗</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QTableWidget" name="AvgIL_Table">
                 <property name="font">
                  <font>
                   <family>等线</family>
                  </font>
                 </property>
                 <property name="rowCount">
                  <number>0</number>
                 </property>
                 <attribute name="horizontalHeaderMinimumSectionSize">
                  <number>20</number>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>94</number>
                 </attribute>
                 <attribute name="verticalHeaderMinimumSectionSize">
                  <number>25</number>
                 </attribute>
                 <attribute name="verticalHeaderDefaultSectionSize">
                  <number>25</number>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="Att_Label">
              <property name="font">
               <font>
                <family>等线</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>带外抑制</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QTableWidget" name="Att_Table">
                 <property name="font">
                  <font>
                   <family>等线</family>
                  </font>
                 </property>
                 <property name="rowCount">
                  <number>0</number>
                 </property>
                 <attribute name="horizontalHeaderMinimumSectionSize">
                  <number>20</number>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>94</number>
                 </attribute>
                 <attribute name="verticalHeaderMinimumSectionSize">
                  <number>25</number>
                 </attribute>
                 <attribute name="verticalHeaderDefaultSectionSize">
                  <number>25</number>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,3">
            <item>
             <widget class="QGroupBox" name="Menu_Label">
              <property name="font">
               <font>
                <family>等线</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Menu</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,0">
               <item>
                <widget class="QGroupBox" name="Marker_Label">
                 <property name="title">
                  <string>Marker</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QPushButton" name="AddButton">
                    <property name="text">
                     <string>Add</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="DeleteButton">
                    <property name="text">
                     <string>Delete</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="HelpButton">
                    <property name="text">
                     <string>Help</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="Temp_Label">
                 <property name="title">
                  <string>TempDiff</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,1,1,1,1,0">
                  <item>
                   <widget class="QGroupBox" name="ValueLabel">
                    <property name="title">
                     <string>测试值/dB</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_11">
                     <item>
                      <widget class="QLineEdit" name="ValueEdit">
                       <property name="text">
                        <string>18</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="LowButton">
                    <property name="text">
                     <string>Low</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="NormalButton">
                    <property name="text">
                     <string>Normal</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="HighButton">
                    <property name="text">
                     <string>High</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="CalculateButton">
                    <property name="text">
                     <string>Calculate</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="TempDiff_Text">
                    <property name="title">
                     <string>温漂</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                     <item>
                      <widget class="QLabel" name="LowText">
                       <property name="text">
                        <string>低温：0.0MHz</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="HighText">
                       <property name="text">
                        <string>高温：0.0MHz</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="File_Label">
              <property name="font">
               <font>
                <family>等线</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>File</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7" stretch="2,2,0,0,2,1">
               <item>
                <widget class="QPushButton" name="ImportButton">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Import</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="ExportButton">
                 <property name="text">
                  <string>Export</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="OpenButton">
                 <property name="text">
                  <string>Open</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="SaveButton">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="ApplyButton">
                 <property name="text">
                  <string>Apply</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="FileName_label">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="midLineWidth">
                  <number>5</number>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Grap_tab">
       <attribute name="title">
        <string>Graphics</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <layout class="QVBoxLayout" name="Graphics_Layout"/>
        </item>
        <item>
         <widget class="QGroupBox" name="Limit_Table">
          <property name="title">
           <string>LimitLine</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <widget class="QTableWidget" name="tableWidget"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ASM_Table">
       <attribute name="title">
        <string>ASM</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_17" stretch="1,3,1">
        <item>
         <widget class="QGroupBox" name="Option_Label">
          <property name="title">
           <string>Option</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_18">
             <item>
              <widget class="QGroupBox" name="Init_Label">
               <property name="title">
                <string>Init</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_20">
                <item>
                 <widget class="QRadioButton" name="IdentityButton">
                  <property name="text">
                   <string>Identity</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="PartialButton">
                  <property name="text">
                   <string>Partial derivative</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_19">
             <item>
              <widget class="QGroupBox" name="Update_Label">
               <property name="title">
                <string>Update</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_21">
                <item>
                 <widget class="QRadioButton" name="UnitButton">
                  <property name="text">
                   <string>Unit Mapping</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="BroydenButton">
                  <property name="text">
                   <string>Broyden</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="JacobiButton">
                  <property name="text">
                   <string>Jacobi</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="Paramter_Label">
          <property name="title">
           <string>Parameter</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,1">
           <item>
            <widget class="QTableWidget" name="Parameter_Table">
             <property name="rowCount">
              <number>11</number>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>Xc Optimal</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Xc 1</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_23" stretch="0">
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_24">
                <item>
                 <widget class="QGroupBox" name="Row_Label">
                  <property name="title">
                   <string>Row</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_25">
                   <item>
                    <widget class="QPushButton" name="RowAddButton">
                     <property name="text">
                      <string>Add</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="RowDeleteButton">
                     <property name="text">
                      <string>Delete</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="Column_Label">
                  <property name="title">
                   <string>Column</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_26">
                   <item>
                    <widget class="QPushButton" name="ColumnAddButton">
                     <property name="text">
                      <string>Add</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="ColumnDeleteButton">
                     <property name="text">
                      <string>Delete</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="ParameterApplyButton">
                  <property name="text">
                   <string>Apply</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Current Status:</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_27">
             <item>
              <widget class="QLabel" name="Status_Label">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
